paper/paper.md

```diff
@@ -0,0 +1,127 @@
+---
+title: '`Osier`: A Python package for multi-objective energy system optimization'
+tags:
+    - Python
+    - energy systems
+    - genetic algorithms
+    - multi-objective optimization
+authors:
+    - name: Samuel G. Dotson
+      orcid: 0000-0002-8662-0336
+      affiliation: 1 
+      corresponding: true
+affiliations:
+    - name: Felix T. Adler Fellow, Department of Nuclear, Plasma, and Radiological Engineering, University of Illinois Urbana-Champaign, USA
+      index: 1
+date: 20 February 2024
+bibliography: paper.bib
+---
+
+# Summary
+Transitioning to a clean energy economy will require expanded energy
+infrastructure. An equitable, or just, transition further requires the
+recognition of the people and communities directly affected by this transition.
+However, public preferences may be ignored during decision-making processes
+related energy infrastructure due to a lack of technical rigor or expertise
+[@johnson:2021]. This challenge is further complicated by the fact that people
+have and express preferences over many dimensions simultaneously.
+Multi-objective optimization offers a method to help decision makers and
+stakeholders understand the problem and analyze tradeoffs among solutions
+[@liebman:1976]. Although, to date, no multi-objective energy modelling
+frameworks exist. Open-source multi-objective energy system framework (`osier`)
+is a Python package for designing and optimizing energy systems across an
+arbitrary number of dimensions. `osier` was designed to help localized
+communities articulate their energy preferences in a technical manner without
+requiring extensive technical expertise. In order to facilitate more robust
+tradeoff analysis, `osier` generates a set of  technology portfolios, called a
+Pareto front, with multi-objective optimization using evolutionary algorithms.
+`osier` also extends the common
+modelling-to-generate-alternatives (MGA) algorithm into many dimensions,
+allowing users to investigate the near-optimal for appealing alternative
+solutions. In this way, `osier` may aid modelers in addressing procedural and
+recognition justice.
+
+# Statement of Need
+There are myriad open- and closed-source energy system optimization models
+(ESOMs) available [@pfenninger:2022]. ESOMs can be used for a variety of tasks
+but are most frequently used for prescriptive analyses meant to guide
+decision-makers in planning processes. However, despite the many available
+models, all of these tools share a fundamental characteristic: Optimization over
+a single economic objective (e.g., total cost or social welfare).
+Simultaneously, there is growing awareness of energy justice and calls for its
+inclusion in energy models [@pfenninger:2014; @vagero:2023]. Some studies
+attempted to incorporate local preferences into energy system design through
+multi-criteria decision analysis (MCDA) and community focus groups
```

Comment:
What is the difference between MCDA and a multi objective energy modeling tool?

Reply:
MCDA focuses more on deciding among alternatives -- the way the alternatives are produced could be through MOO but they could also be generated through specific scenario analysis. At least, this is my understanding of the literature.